### PR TITLE
Update /partners/gsi

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Update [/partners/gsi](https://canonical-com-901.demos.haus/partners/gsi) according to [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9452-22656&t=3C6aCRgYhLX0PwlP-0)
- Extended the logo partial

## QA

- Compare [/partners/gsi](https://canonical-com-901.demos.haus/partners/gsi) against [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9452-22656&t=3C6aCRgYhLX0PwlP-0)

## Issue / Card

Fixes WD-3743

### CODE REVIEW

File: templates/partners/gsi.html [hunk 1]
```diff
@@ -6,93 +6,97 @@
...
-      <div class="p-logo-section__items u-no-margin--bottom">
-        {% for partner in partners %}
-        <div class="p-logo-section__item">
-          <img class="p-logo-section__logo" src="{{ partner.logo }}" style="min-width: 5rem;" alt="{{ partner.slug}}">
-        </div>
-        {% endfor %}
+    <div class="col">
+      <div class="p-block">
+        <p>Canonical works with Global System Integrators around the world to build solutions that help our joint customers be successful in their business.</p>
+        <p>Together we've successfully helped telecom providers, governmental agencies, financial services companies and other industry leaders overcome their toughest technological challenges. We help Global System Integrators build solutions and platforms on top of open, secure, reliable and professionally-supported foundations. </p>
```
Comment (mtruj013) [inline templates/partners/gsi.html:25]: From the design, looks like the last sentence should be in it's own `<p>`

File: templates/partners/gsi.html [hunk 2]
```diff
@@ -6,93 +6,97 @@
...
     </div>
   </div>
 </section>
-{% endif %}
 
-<section class="p-strip--light is-deep">
-  <div class="u-fixed-width">
-    <h2>Building powerful customer solutions together</h2>
-  </div>
+{% with title="Meet some of our global system integrator partners", link_text="See all of our gsi partners", link_href="/partners/find-a-partner?filters=global-system-integrators", is_split=True %}
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/partners/gsi.html:32]: The link is right aligned in the mock-up

File: templates/partners/gsi.html [hunk 3]
```diff
@@ -169,12 +153,11 @@ <h2>
         </li>
       </ul>
     </div>
-    <div class="col-5 u-hide--small u-hide--medium u-align--center u-vertically-center">
-      <img src="https://assets.ubuntu.com/v1/5d8aa78b-1+Discovery.svg" width="335" height="200" alt="">
-    </div>
   </div>
 </section>
 
+{% include '/partners/partial/_co-sell-with-canonical.html' %}
```
Comment (mtruj013) [inline templates/partners/gsi.html:159]: Ah! I see now, yeah makes sense to make this a partial for sure, but I may be missing some context here. Is the copy supposed to be "Access our partner portal" or "Co-sell with Canonical through the partner portal"? And I assume the image in the mock-up is just a placeholder and not meant to be included yet, is that right?

**Review by mtruj013 — APPROVED** [review]
LGTM, just some questions and minor suggestions

File: templates/partners/gsi.html (hunk 3 above)
Comment (ClementChaumel) [reply templates/partners/gsi.html:159]: I'm not too sure but last time there was such an image we ended up not including it so I'm just applying the same principles here!

File: templates/partners/gsi.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (ClementChaumel) [reply templates/partners/gsi.html:32]: The copy in the design says "A selection  of our Public Cloud partners" I assume it's a carry over from designing the public cloud page so I just ignored it and kept the current copy. Good call on the capitalisation!

**Comment by juanruitina** [comment]
- [] Below the logos, please change the link to "See all GSI partners" and align it to the right.
- [] Remove the checkbox bullets from the list under "Learn more about our solutions", just keep links and borders.
- [] Please add [Atos's logo](https://assets.ubuntu.com/v1/b892a19b-atos-logo.png) to the list. I'm double checking it's the right call, but you can incorporate it already.

Other than that, LGTM!

**Comment by ClementChaumel** [comment]
@juanruitina 

> * [] Below the logos, please change the link to "See all GSI partners" and align it to the right.

That's the way it used to be but @lyubomir-popov changed the alignment to this for the IHV OEM page and i'd like to keep it consistent.

> * [] Please add [Atos's logo](https://assets.ubuntu.com/v1/b892a19b-atos-logo.png) to the list. I'm double checking it's the right call, but you can incorporate it already.
> 

I can't just do it like that, we'd need to go to the admin view and tick the "Featured" box. I'd rather not do it myself and instead have the person in charge of the Atos partnership agree to it first.
image

**Comment by lyubomir-popov** [comment]
@ClementChaumel Did this happen here? 
image

**Comment by lyubomir-popov** [comment]
Logo missing (6 vs 5), link alignment: 
image

**Comment by lyubomir-popov** [comment]
Can we please break this into two paragraphs?

image


And finally, the logos need resizing, but I undersand we'll do a sweep for poorly cropped logos on the feature branch once everything is in, right?

File: templates/partners/gsi.html [hunk 4]
```diff
@@ -6,93 +6,98 @@
...
+    <hr class="p-rule">
+  </div>
+  <div class="row--50-50 p-block">
+    <div class="col">
+      <h2>Building powerful customer solutions together</h2>
     </div>
-    <div class="col-5 col-medium-2 u-hide--small u-align--center u-vertically-center">
-      <img src="https://assets.ubuntu.com/v1/58442bcb-business-promo-webinar.png" width="200" height="125" alt="">
+    <div class="col">
+      <p>Join our webinars created specifically for Global System Integrators who are overcoming difficult technological challenges to build solutions and platforms for their customers. We will talk about scaleable, automated and repeatable deployments that will help your customers meet their digital transformation needs.</p>
```
Comment (lyubomir-popov) [inline templates/partners/gsi.html:46]:
```suggestion
      <p>Join our webinars created specifically for Global System Integrators who are overcoming difficult technological challenges to build solutions and platforms for their customers. </p><p>We will talk about scaleable, automated and repeatable deployments that will help your customers meet their digital transformation needs.</p>
```

**Review by lyubomir-popov — APPROVED** [review]
Looks good, one suggestion to break up a paragraph in the code. The logos we can finalise in the feature branch so +1 to this PR from me.

**Pull request merged**